style.css

```diff
@@ -0,0 +1,191 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  min-height: 100vh;
+
+  background: #131313;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.container {
+  position: relative;
+}
+.container .card {
+  position: relative;
+  width: 320px;
+  height: 450px;
+
+  background: #232323;
+
+  border-radius: 20px;
+  overflow: hidden;
+}
+.container .card:before {
+  width: 100%;
+  height: 100%;
+
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+
+  background: #9bdc28;
+  clip-path: circle(150px at 80% 20%);
+
+  transition: 0.5s ease-in-out;
+}
+.container .card:hover:before {
+  clip-path: circle(300px at 80% -20%);
+}
+.container .card:after {
+  content: "Nike";
+  position: absolute;
+  top: 150px;
+  left: -20px;
+
+  font-size: 10em;
+  font-weight: 800;
+  font-style: italic;
+  text-transform: uppercase;
+  color: rgba(255, 255, 255, 0.04);
+}
+.container .card .imgbx {
+  width: 100%;
+  height: 220px;
+
+  position: absolute;
+  top: 50%;
+
+  transform: translateY(-50%);
+  z-index: 1000;
+
+  transition: 0.5s;
+}
+.container .card:hover .imgbx {
+  top: 0%;
+  transform: translateY(0%);
+  transform: translateX(12%);
+}
+.container .card .imgbx img {
+  width: 270px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%) rotate(25deg);
+}
+.container .card .contentbx {
+  width: 100%;
+  height: 100px;
+
+  position: absolute;
+  bottom: 0;
+
+  text-align: center;
+  transition: 1s;
+  z-index: 10;
+}
+.container .card:hover .contentbx {
+  height: 210px;
```

Comment:
always avoid setting height, when we place things inside container height automatically gets adjusted